src/components/src/test/groovy/org/apache/jmeter/visualizers/backend/influxdb/InfluxDBRawBackendListenerClientSpec.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jmeter.visualizers.backend.influxdb
+
+import org.apache.jmeter.samplers.SampleResult
+import org.apache.jmeter.visualizers.backend.BackendListenerContext
+
+import spock.lang.Specification
+
+class InfluxDBRawBackendListenerClientSpec extends Specification {
+
+    def sut = new InfluxDBRawBackendListenerClient()
+    def defaultContext = new BackendListenerContext(sut.getDefaultParameters())
+
+    def createOkSample() {
+        def now = System.currentTimeMillis()
```

Comment:
Is `currentTimeMillis` needed here?
Can a constant be used instead?

Reply:
Could be, what would be the advantage of using say 1 or 1585413149344, vs. this call?

Reply:
Using a fixed value in test helps to keep the test reproducible, and it helps to avoid unexpected timezone issues.

If you want to exercise timezone issues, please create a separate test for that.

Reply:
Updated.